### ISSUE TITLE
Possibility to set problem criticality to "Warning" or "Info"

### ISSUE THREAD

**Issue by SecMS** [znck/grammarly#332]
Hi there,

cool plugin. However, it totally spams my problem view in vscode. 
Could you please add the feature that one can select whether grammerly problems are considered to be "Errors", "Warnings", or "Infos".

Many thanks in advance and cheers,
Mike

**Comment by silopolis** [comment]
Probably the most "annoying" issue until the permanent dismiss/ignore rules aren't available.

**Comment by Babalion** [comment]
I agree absolutely. Would be nice to configure the severity level in the options.

**Comment by yttriumz** [comment]
Once upon a time there was a `grammarly.severity` option ([line 271 in `package.json`](https://github.com/znck/grammarly/pull/73/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)) to control the diagnostic severity. Not sure why it's gone.
